tools/azsdk-cli/Azure.Sdk.Tools.Cli/Tools/APIView/APIViewRevisionTool.cs

```diff
@@ -0,0 +1,282 @@
+using System.CommandLine;
+using System.CommandLine.Invocation;
+using System.ComponentModel;
+using System.Text.Json;
+using Azure.Sdk.Tools.Cli.Commands;
+using Azure.Sdk.Tools.Cli.Models;
+using Azure.Sdk.Tools.Cli.Models.APIView;
+using Azure.Sdk.Tools.Cli.Models.Responses;
+using Azure.Sdk.Tools.Cli.Services;
+using ModelContextProtocol.Server;
+
+namespace Azure.Sdk.Tools.Cli.Tools.APIView;
+
+[McpServerToolType]
+[Description("APIView revision operations including comments and content")]
+public class APIViewRevisionTool : MCPMultiCommandTool
+{
+    public override CommandGroup[] CommandHierarchy { get; set; } = [
+        SharedCommandGroups.APIView,
+        SharedCommandGroups.APIViewRevision
+    ];
+
+    private static readonly string[] validContentTypes = ["Text", "CodeFile"];
+    private static readonly string[] validRevisionSelectionTypes = ["Latest", "LatestApproved", "LatestAutomatic", "LatestManual", "Undefined"];
+
+    private readonly IAPIViewService _apiViewService;
+    private readonly ILogger<APIViewRevisionTool> _logger;
+
+    private readonly Option<string> environmentOption = new("--environment",
+        description: "The APIView environment (defaults to production)", getDefaultValue: () => "production");
+
+    private readonly Option<string> outputFileOption = 
+        new("--output-file", "Output file path to save the revision content");
+
+    private readonly Option<string> contentReturnTypeOption = new("--content-return-type",
+        description: "The APIView revision content type (text or codefile). Defaults to 'text'.", getDefaultValue: () => "text");
+    private readonly Option<string> revisionSelectionTypeOption = new("--revision-selection-type",
+        description: "The type of revision selection (Latest, latestApproved, LatestAutomatic, LatestManual)");
```

Comment:
If this option is required only when review ID option is used, then it's better to mention in the description.